### PR TITLE
Day six

### PR DESCRIPTION
**Pull request by Talaspaev**
image

image

### CODE REVIEW

File: src/Components/Dialogs/GetPerson/getPerson.jsx [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
 
 const GetPerson = (props) => {
-  const { items } = props;
+  const { items, Path } = props;
   const itemsArray = Object.values(items);
 
-  const sd = ({ id, name }) => (<div key={id}><NavLink to={`${Path.dialog}/${id}`} className={style.item}>{name}</NavLink></div>);
+  const sd = ({ id, name, src }) => (
+    <div key={id} className={style.wrapper}>
+      <div className={style.imgContainer}><img src={src} className={style.img} alt="alt" onError="this.onerror=null;" /></div>
```
Comment (Dmitry-White) [inline src/Components/Dialogs/GetPerson/getPerson.jsx]: Consider moving the onError handler into a separate dedicated function in the current component.

File: src/redux/state.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    { id: 1, name: 'Dymuch', src: 'https://upload.wikimedia.org/wikipedia/commons/9/9f/Dmitry_Medvedev_2016.jpg' },
+    { id: 2, name: 'Andrey', src: 'https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcQ3ouc4dEwInAn-fzd0par_bEkPsHUxzx4J-nXg1sk-SC4ABqAt' },
+    { id: 3, name: 'Natalya', src: 'https://images11.cosmopolitan.ru/upload/img_cache/7b3/7b3b3da698f08bee6e9e7402053f6df5_ce_750x497x0x129_fitted_740x0.jpg' },
+    { id: 4, name: 'Igor', src: 'https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRNtwQUeVJ4vImAb6HiD2qFiEXchzJJKPDWUU96D33THFtBskw1' },
+    { id: 5, name: 'Adil', src: 'https://img.gazeta.ru/files3/784/11560784/skrskr2-pic905-895x505-4108.jpg' }],
+  friends: [
+    { id: 1, name: 'Dymuch', src: 'https://upload.wikimedia.org/wikipedia/commons/9/9f/Dmitry_Medvedev_2016.jpg' },
+    { id: 2, name: 'Natalya', src: 'https://images11.cosmopolitan.ru/upload/img_cache/7b3/7b3b3da698f08bee6e9e7402053f6df5_ce_750x497x0x129_fitted_740x0.jpg' },
+    { id: 3, name: 'Adil', src: 'https://img.gazeta.ru/files3/784/11560784/skrskr2-pic905-895x505-4108.jpg' }],
+  Path: {
```
Comment (Dmitry-White) [inline src/redux/state.js]: This does not belong to state in the current architecture.

File: src/redux/state.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    dialog: 'dialog',
+  },
+  I18Y: {
```
Comment (Dmitry-White) [inline src/redux/state.js]: This does not belong to state in the current architecture.

File: src/Components/SiteBar/siteBar.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import style from './siteBar.module.css';
+
+const SiteBar = (props) => {
+  const { friends } = props.state;
+  const st = Object.values(friends);
```
[the remark this answers is not in the document]
Comment (Talaspaev) [reply src/Components/SiteBar/siteBar.js]: Done

File: src/Components/Dialogs/GetPerson/getPerson.jsx [hunk 5]
```diff
@@ -1,15 +1,18 @@
...
+      <NavLink to={`${Path.dialog}/${id}`} className={style.item}>{name}</NavLink>
```
[the remark this answers is not in the document]
Comment (Talaspaev) [reply src/Components/Dialogs/GetPerson/getPerson.jsx]: done

File: src/Components/Profile/MyPosts/myPosts.jsx [hunk 6]
```diff
@@ -2,21 +2,22 @@ import React from 'react';
...
-  <div>
-    My posts
-    <div />
-    <textarea />
-    <button type="button">Add post</button>
-    <div className={style.posts}>
-      <Post message={I18Y.qus} />
-      <Post message={I18Y.ans} />
+const MyPosts = (props) => {
+  const { state } = props;
```
[the remark this answers is not in the document]
Comment (Talaspaev) [reply src/Components/Profile/MyPosts/myPosts.jsx]: I returned

File: src/Components/Profile/Profile.jsx [hunk 7]
```diff
@@ -3,11 +3,14 @@ import React from 'react';
...
 import ProfileInfo from './ProfileInfo/profileInfo';
 
-const Profile = () => (
-  <div>
-    <ProfileInfo />
-    <MyPosts />
-  </div>
-);
+const Profile = (props) => {
+  const { I18Y } = props.state;
```
[the remark this answers is not in the document]
Comment (Talaspaev) [reply src/Components/Profile/Profile.jsx]: I returned

File: src/Components/Dialogs/dialogs.jsx [hunk 8]
```diff
@@ -3,20 +3,21 @@ import React from 'react';
...
 
-const Dialogs = () => (
-  <div className={style.dialogs}>
-    <div className={style.dialogsItems}>
-      <GetPerson items={dialogsData} />
-    </div>
-    <div className={style.dialogsMessage}>
-      <GetItem items={messagesData} />
+const Dialogs = (props) => {
+  const { dialogsData, messagesData, Path } = props.state;
```
[the remark this answers is not in the document]
Comment (Talaspaev) [reply src/Components/Dialogs/dialogs.jsx]: I moved ROUTES to constant.js

File: src/Components/Dialogs/GetPerson/getPerson.jsx (hunk 1 above)
Comment (Talaspaev) [reply src/Components/Dialogs/GetPerson/getPerson.jsx]: Okey.i created

File: src/redux/state.js (hunk 2 above)
Comment (Talaspaev) [reply src/redux/state.js]: Sure

File: src/redux/state.js (hunk 3 above)
Comment (Talaspaev) [reply src/redux/state.js]: Sure

File: src/Components/Dialogs/dialogs.jsx [hunk 9]
```diff
@@ -3,20 +3,23 @@ import React from 'react';
...
-const Dialogs = () => (
-  <div className={style.dialogs}>
-    <div className={style.dialogsItems}>
-      <GetPerson items={dialogsData} />
+const Dialogs = (props) => {
+  const { dialogsData, messagesData } = props.state;
+  return (
+    <div className={style.dialogs}>
+      <div className={style.dialogsItems}>
+        <GetPerson items={dialogsData} ROUTES={ROUTES} />
```
Comment (Dmitry-White) [inline src/Components/Dialogs/dialogs.jsx]: Consider NOT naming the props in uppercase.

Comment (Talaspaev) [reply src/Components/Dialogs/dialogs.jsx]: Yes.sure I made a mistake

**Pull request merged**